Tools/Scripts/libraries/webkitbugspy/webkitbugspy/github.py

```diff
@@ -434,6 +434,10 @@ def set(self, issue, assignee=None, opened=None, why=None, project=None, compone
             sys.stderr.write('GitHub does not support state at this time\n')
             return None
 
+        if see_also:
+            sys.stderr.write('GitHub does not support the see_also field at this time\n')
```

Comment:
I'll note that I don't like the fact we just write to `sys.stderr` and then return `None` in what's kinda meant to be a library, but it is nothing but following the status quo.